dotcom-rendering/src/web/components/LiveBlock.tsx

```diff
@@ -245,80 +170,46 @@ export const LiveBlock = ({
 		block.blockLastUpdated > block.blockFirstPublished;
 
 	return (
-		<Container id={block.id} palette={palette}>
+		<LiveBlockContainer
+			id={block.id}
+			borderColour={palette.border.liveBlock}
+		>
 			<Header>
-				<aside
-					css={css`
-						${from.phablet} {
-							/* Yes, we do need both */
-							min-width: ${ASIDE_WIDTH + GUTTER}px;
-							width: ${ASIDE_WIDTH + GUTTER}px;
-						}
-						padding-left: ${GUTTER}px;
-						padding-right: ${GUTTER}px;
-						padding-bottom: ${space[2]}px;
-					`}
-				>
-					{block.blockFirstPublished && (
-						<FirstPublished
-							firstPublished={block.blockFirstPublished}
-							blockLink={blockLink}
-						/>
-					)}
-				</aside>
-				<span>
-					{block.title && <BlockTitle title={block.title} />}
-					{headerElement &&
-						renderArticleElement({
-							format,
-							palette,
-							element: headerElement,
-							isMainMedia: false,
-							host,
-							adTargeting,
-							index: 0,
-							pageId,
-							webTitle,
-						})}
-				</span>
+				{block.blockFirstPublished && (
+					<FirstPublished
+						firstPublished={block.blockFirstPublished}
+						blockLink={blockLink}
+					/>
+				)}
+				{block.title && <BlockTitle title={block.title} />}
+				{headerElement &&
+					renderArticleElement({
+						format,
+						palette,
+						element: headerElement,
+						isMainMedia: false,
+						host,
+						adTargeting,
+						index: 0,
+						pageId,
+						webTitle,
+					})}
 			</Header>
-			<main>
-				{/* For each element, we decide what margins to set depending on the type */}
-				{mainElements.map((element, index) => {
-					if (typesWeStretch.includes(element._type)) {
-						return (
-							<BlockMedia key={`${element._type}-${index}`}>
-								{renderArticleElement({
-									format,
-									palette,
-									element,
-									adTargeting,
-									host,
-									index,
-									isMainMedia: false,
-									pageId,
-									webTitle,
-								})}
-							</BlockMedia>
-						);
-					}
-
-					return (
-						<BlockText key={`${element._type}-${index}`}>
-							{renderArticleElement({
-								format,
-								palette,
-								element,
-								isMainMedia: false,
-								index,
-								pageId,
-								webTitle,
-							})}
-						</BlockText>
-					);
-				})}
-			</main>
-			<Footer>
+			{/* For each element, we decide what margins to set depending on the type */}
```

Comment:
Is this comment still true?

Reply:
I guess not, given the container is handling the padding now 👍🏻